src/scipp/core/binning.py

```diff
@@ -95,6 +98,13 @@ def make_binned(x: _cpp.DataArray,
         groups = []
     if edges is None:
         edges = []
+    if isinstance(x, Variable):
+        coords = edges + groups
+        if len(coords) != 1:
+            raise ValueError("Edges for exactly one dimension must be specified when "
+                             "binning or histogramming a variable.")
+        data = scalar(1.0, unit='counts').broadcast(sizes=x.sizes).copy()
```

Comment:
Is this copy required?

Reply:
Yes, because `bin` does not work with columns that are broadcast (not stride 1).